playbook.yml

```diff
@@ -80,12 +82,12 @@
       args:
         chdir: /lib
       with_items:
-        - git clone https://github.com/UKHomeOffice/data-transfer
+        - test -e /lib/data-transfer || git clone https://github.com/UKHomeOffice/data-transfer
```

Comment:
I'm not sure this will work? have you tested it?

Reply:
http://docs.ansible.com/ansible/latest/git_module.html

Reply:
it does, but was for running the playbook multiple times so not necessarily needed - I will refactor to use the ansible git module too